setup.py

```diff
@@ -0,0 +1,22 @@
+"""@package dynamic_graph_manager
+
+@file setup.py
+@author Maximilien Naveau (maximilien.naveau@gmail.com)
+@license License BSD-3-Clause
+@copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+@date 2019-05-22
+
+@brief This file defines the python modules in this package.
+
+"""
+
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+# fetch values from package.xml
+setup_args = generate_distutils_setup(
+    packages=['mpi_cmake_modules'],
+    package_dir={'': 'scripts'},
```

Comment:
Ideally Python scripts and modules should be separated.  The former should be in a directory "scripts" while the latter should be in "python".  In the setup.py only the "python" directory needs to be mentioned.

Example structure:
```
package root
├── python
│   └── package_name
│       ├── __init__.py
│       └── module.py
├── scripts
│   └── executable_script.py
├── CMakeLists.txt
├── package.xml
└── setup.py
```